DVSRegister/Views/Consent/PartialViews/_ServiceDetailsView.cshtml

```diff
@@ -69,32 +69,46 @@
 
 
     @if (Model.HasGPG44 && Model.ServiceQualityLevelMapping != null)
-    {
-        <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">
-                GPG44 quality of authenticator
-            </dt>
-            <dd class="govuk-summary-list__value">
-                @foreach (var item in Model.ServiceQualityLevelMapping.Where(x => x.QualityLevel.QualityType == QualityTypeEnum.Authentication))
-                {
-                    <div>@item.QualityLevel.Level</div>
-                }
-            </dd>
-        </div>
-
-
-
-        <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">
-                GPG44 level of protection
-            </dt>
-            <dd class="govuk-summary-list__value">
-                @foreach (var item in Model.ServiceQualityLevelMapping.Where(x => x.QualityLevel.QualityType == QualityTypeEnum.Protection))
-                {
-                    <div>@item.QualityLevel.Level</div>
-                }
-            </dd>
-        </div>
+        {
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                    GPG44 quality of authenticator
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    @if (Model.ServiceQualityLevelMapping.Any(x => x.QualityLevel.QualityType == QualityTypeEnum.Authentication))
```

Comment:
need to add Model.ServiceQualityMapping != null and ServiceQualityMapping .any()  Any will check the cpount and if ServiceQualityMapping  is null will throw null reference exeption